CHANGELOG.rst

```diff
@@ -1,10 +1,17 @@
 Changes by Version
 ==================
 
-0.24.1 (unreleased)
+0.25.0 (2018-03-07)
 -------------------
 
-- Nothing yet.
+- Consolidate codec exception handling and logs exceptions (#362) <Tomasz Adamski>
+- Deprecated StatsReporter (#349) <Juraci Paixão Kröhling>
```

Comment:
Should https://github.com/jaegertracing/jaeger-client-java/commit/cf99e756624b6bd8b04df4f2bd3a6b9137fa3adc be included here?